README.md

```diff
@@ -477,6 +477,20 @@ colorful_winsep = {
 </tr>
 <!-- colorful_winsep.nvim -->
 
+<!-- dadbod-ui -->
+</tr>
+<tr>
+<td> <a href="https://github.com/kristijanhusak/vim-dadbod-ui">dadbod-ui</a> </td>
+<td>
+
+```lua
+dadbod_ui = true
```

Comment:
I guess `false` actually, as well as the vimdoc below.

Reply:
Updated in [5b382d4](https://github.com/catppuccin/nvim/pull/747/commits/5b382d402dd30ad0a2f94400853e22a04595e1a1)